app/code/core/Mage/Adminhtml/Block/Catalog/Category/Attribute.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
```

Comment:
You have forgotten the copy comment (you have added it elsewhere).

Reply:
are you referring about the copyright? I was told by @Flyingmana (it's the comment above yours) to remove it because new files (it they're not copied from old ones) shouldn't have the old copyright

Reply:
are you referring about the copyright? I was told by @Flyingmana (it's the comment above yours) to remove it because new files (it they're not copied from old ones) shouldn't have the old copyright

Reply:
oh... ok.